### ISSUE TITLE
Add a table with a registers

### ISSUE THREAD

**Issue by vazhnov** [johnwinans/rvalp#3]
Thank you for a book!

What do you think about to add a table like here https://bronzebeard.readthedocs.io/en/latest/assembly_language.html#registers (MIT license) into your chapter "1.1.2.3 Registers"?

P.S.: If yes, I would recommend to expand this table with `pc` register.

**Comment by johnwinans** [comment]
Yeah, the intro chapter needs work.  Maybe a table in there _would_ help.
I guess I could copy the table on page 58?
I suppose I could add an extended box to the bottom showing the PC.

**Comment by vazhnov** [comment]
Oops, I see this table in "5.4 CPU Registers", on the page 58. Sorry :)
But `pc` register can be added into this table, with the same alias.
